src/main.rs

```diff
@@ -14,21 +14,20 @@ struct Arguments {
     pub log_filter: String,
     #[structopt(long, env = "BIND_ADDRESS", default_value = "0.0.0.0:8080")]
     bind_address: SocketAddr,
-    #[structopt(long, env = "DUNE_DATA_FILE", default_value = "./user_data.json")]
-    dune_data_file: String,
+    #[structopt(long, env = "DUNE_DATA_FOLDER", default_value = "./data/dune_data/")]
+    dune_data_folder: String,
 }
 
 #[tokio::main]
 async fn main() {
     let args = Arguments::from_args();
     initialize(args.log_filter.as_str());
     tracing::info!("running data-server with {:#?}", args);
-    let dune_download_file = args.dune_data_file;
-
+    let dune_download_folder = args.dune_data_folder;
```

Comment:
For coming changes, its better to work with the folder. The folder will contain serveral subpath with the app data and the user_data